### ISSUE TITLE
tutorials for trained the model from scratch for Cell-type Annotation

### ISSUE THREAD

**Issue by yueming-ding** [bowang-lab/scGPT#51]
Hi we are very interested in how you trained the model from scratch for Cell-type Annotation. Could you share your codes/tutorials if possible? Thanks!

**Comment by subercui** [comment]
Hi, thank you for your interest! The training from scratch can also be readily supported using the same tutorial [notebook](https://github.com/bowang-lab/scGPT/blob/main/tutorials/Tutorial_Annotation.ipynb). 

In the following cell the notebook:

![image](https://github.com/bowang-lab/scGPT/assets/11674033/015bb881-d141-4e56-9d18-39251213fd4b)

Set `load_model=None` then the model loading step would be skipped. Note that we did find it easier to train a smaller model from scratch rather than a large one. For this reason, the default model layer_size and nlayers are set to small values in the screenshot. You may simply follow these settings.

**Comment by qiliu-ghddi** [comment]
Hello, we are attempting to replicate the Cell-type Annotation as described in the paper. In our effort to train the model from scratch, I have set the `load_model` parameter to `None` and have reduced both the `layer_size` and `nlayers`. but the best results on the Multiple Sclerosis dataset we got is `Accuracy: 0.592, Precision: 0.381, Recall: 0.370, Macro F1: 0.328`.
which had a significant discrepancy in results compared to those reported. 

![Snipaste_2024-01-12_15-33-25](https://github.com/bowang-lab/scGPT/assets/87366652/21cb90a3-2acc-4361-bc0b-1328713a4c35)

![Snipaste_2024-01-12_15-38-46](https://github.com/bowang-lab/scGPT/assets/87366652/3984973a-192f-424e-a18b-31de6eaa7ccb)


Could you please provide us with the exact parameter settings you used in the paper for training the model from scratch? Details on the architecture, learning rate, number of epochs, and any other relevant hyperparameters would be highly beneficial.

![Snipaste_2024-01-12_15-29-51](https://github.com/bowang-lab/scGPT/assets/87366652/6565e6e5-f740-4af8-b115-1e0dfbd5d44c)

This information will greatly aid in understanding the nuances of the training process and improve the reproducibility of the results. Thank you for your time and assistance.
